### PR TITLE
Add Edit User permission to permissions grid

### LINKED ISSUE #1845: Missing control for permission user.edit in the admin
The permission `user.edit` which control which groups can edit a user is supported in Flarum and can be set via the API. However it is not present on the Permission page of the admin panel.

### PR DESCRIPTION
**Pull request by tankerkiller125**
**Fixes #1845**

**Changes proposed in this pull request:**
This change adds the User Edit permission to the permissions panel in the administration view

**Reviewers should focus on:**
General code, this is my first pull request for Flarum, the lang-eng repo will also have a pull request that adds the translation for the label

**Screenshot**

Annotation 2019-08-24 210709

**Confirmed**

- [x] Frontend changes: tested on a local Flarum installation.
- [ ] Backend changes: tests are green (run `php vendor/bin/phpunit`).

**Required changes:**
- [x] Related core extension PRs: https://github.com/flarum/lang-english/pull/145

### CODE REVIEW

**Comment by vingrad** [comment]
Thanks for you contributing, but please do not commit the files in the dist directory.

> Flarum's front-end code is written in ES6 and transpiled into JavaScript. During development you will need to recompile the JavaScript using Node.js. Please do not commit the resulting dist files when sending PRs; this is automatically taken care of when changes are merged into the master branch.
Reference: https://flarum.org/docs/contributing.html#development-setup

**Comment by tankerkiller125** [comment]
I've undone the commit and force pushed a new one that only contains the `src` changes, no `dist` files are included with the change now.

File: js/src/admin/components/PermissionGrid.js [hunk 1]
```diff
@@ -228,6 +228,11 @@ export default class PermissionGrid extends Component {
       permission: 'discussion.deletePosts'
     }, 60);
 
+    items.add('userEdit', {
+      icon: 'fas fa-user-cog',
+      label: app.translator.trans('core.admin.permissions.edit_user_label'),
+      permission: 'user.edit'
+    }, 60);
```
Comment (franzliedke) [inline js/src/admin/components/PermissionGrid.js]: One newline after, please. :grin:

**Comment by franzliedke** [comment]
Code looks good, but let's not merge this until the discussion in #1845 has been resolved.

**Comment by franzliedke** [comment]
I am closing this for now, until we have reached a decision in #1845.

@tankerkiller125 Thanks a lot for your contribution! If we decide to move forward with this, we will surely reactivate this. I am just trying to keep the number of open PRs down. I hope you understand. :wink: Have a great weekend!

**Comment by franzliedke** [comment]
@tankerkiller125 Thanks and congrats!

**Pull request merged**